From 26d5cb74b9128dda0656f6750e2d074b06837797 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 22 Jul 2018 17:43:22 +0300
Subject: [PATCH 0813/1200] dts: a7k/a8k: add phy-speed property for all sfi
 comphys

The SFI can work with different speeds, add phy-speed property anywhere
it was missing.

Change-Id: Iab666416d10489055e27f4b1cfba8b7ccaeb8bde
Signed-off-by: Igal Liberman <igall@marvell.com>
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/58244
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 arch/arm/dts/armada-7040-db-C.dts  | 1 +
 arch/arm/dts/armada-8040-db-C.dts  | 2 ++
 arch/arm/dts/armada-8040-db-D.dts  | 1 +
 arch/arm/dts/armada-8040-db-E.dts  | 1 +
 arch/arm/dts/armada-8040-db-F.dts  | 1 +
 arch/arm/dts/armada-8040-db.dts    | 2 ++
 arch/arm/dts/armada-8040-mcbin.dts | 2 ++
 7 files changed, 10 insertions(+)

diff --git a/arch/arm/dts/armada-7040-db-C.dts b/arch/arm/dts/armada-7040-db-C.dts
index e6cafdb7e4..20b6204c92 100644
--- a/arch/arm/dts/armada-7040-db-C.dts
+++ b/arch/arm/dts/armada-7040-db-C.dts
@@ -153,6 +153,7 @@
 
 	phy2 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 
 	phy3 {
diff --git a/arch/arm/dts/armada-8040-db-C.dts b/arch/arm/dts/armada-8040-db-C.dts
index 3940e0ce0b..06c0f92a10 100644
--- a/arch/arm/dts/armada-8040-db-C.dts
+++ b/arch/arm/dts/armada-8040-db-C.dts
@@ -122,6 +122,7 @@
 	};
 	phy2 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy3 {
 		phy-type = <COMPHY_TYPE_SATA1>;
@@ -250,6 +251,7 @@
 	};
 	phy2 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy3 {
 		phy-type = <COMPHY_TYPE_SATA1>;
diff --git a/arch/arm/dts/armada-8040-db-D.dts b/arch/arm/dts/armada-8040-db-D.dts
index c0ed010b32..524edfe336 100644
--- a/arch/arm/dts/armada-8040-db-D.dts
+++ b/arch/arm/dts/armada-8040-db-D.dts
@@ -128,6 +128,7 @@
 	};
 	phy2 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy3 {
 		phy-type = <COMPHY_TYPE_SATA1>;
diff --git a/arch/arm/dts/armada-8040-db-E.dts b/arch/arm/dts/armada-8040-db-E.dts
index 703643e280..30ca2f996e 100644
--- a/arch/arm/dts/armada-8040-db-E.dts
+++ b/arch/arm/dts/armada-8040-db-E.dts
@@ -122,6 +122,7 @@
 	};
 	phy2 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy3 {
 		phy-type = <COMPHY_TYPE_SATA1>;
diff --git a/arch/arm/dts/armada-8040-db-F.dts b/arch/arm/dts/armada-8040-db-F.dts
index a70d32f41e..b8230268d7 100644
--- a/arch/arm/dts/armada-8040-db-F.dts
+++ b/arch/arm/dts/armada-8040-db-F.dts
@@ -297,6 +297,7 @@
 	};
 	phy2 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy3 {
 		phy-type = <COMPHY_TYPE_SATA1>;
diff --git a/arch/arm/dts/armada-8040-db.dts b/arch/arm/dts/armada-8040-db.dts
index 2bbcc94116..12cfeb4431 100644
--- a/arch/arm/dts/armada-8040-db.dts
+++ b/arch/arm/dts/armada-8040-db.dts
@@ -159,6 +159,7 @@
 	};
 	phy2 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy3 {
 		phy-type = <COMPHY_TYPE_SATA1>;
@@ -266,6 +267,7 @@
 	};
 	phy2 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy3 {
 		phy-type = <COMPHY_TYPE_SATA1>;
diff --git a/arch/arm/dts/armada-8040-mcbin.dts b/arch/arm/dts/armada-8040-mcbin.dts
index 6b6e923a13..75fa88e0cc 100644
--- a/arch/arm/dts/armada-8040-mcbin.dts
+++ b/arch/arm/dts/armada-8040-mcbin.dts
@@ -212,6 +212,7 @@
 	};
 	phy4 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy5 {
 		phy-type = <COMPHY_TYPE_SATA1>;
@@ -359,6 +360,7 @@
 	};
 	phy4 {
 		phy-type = <COMPHY_TYPE_SFI0>;
+		phy-speed = <COMPHY_SPEED_10_3125G>;
 	};
 	phy5 {
 		phy-type = <COMPHY_TYPE_SGMII2>;
-- 
2.22.0

